src/controller/ReadInteraction.h

```diff
@@ -94,7 +107,7 @@ CHIP_ERROR ReportAttribute(Messaging::ExchangeManager * exchangeMgr, EndpointId
  */
 template <typename DecodableAttributeType>
 CHIP_ERROR ReadAttribute(Messaging::ExchangeManager * exchangeMgr, const SessionHandle & sessionHandle, EndpointId endpointId,
-                         ClusterId clusterId, AttributeId attributeId,
+                         ClusterId clusterId, AttributeId attributeId, const Optional<DataVersion> & aDataVersion,
```

Comment:
Again, put this at the end, with a default of NullOptional, because most consumers right now don't care about it...

Reply:
fully agree on it, updated, thanks